ci-operator/step-registry/openshift/e2e/aws/loki/openshift-e2e-aws-loki-workflow.yaml

```diff
@@ -6,6 +6,7 @@ workflow:
     - chain: ipi-aws-pre
     test:
     - ref: openshift-e2e-test
+    - ref: machine-api-e2e-test
```

Comment:
I'd prefer a separate workflow for that, as `openshift-e2e-aws-loki` can be used by other projects

Reply:
Danil and i talked about this, and we think we can just remove it from the loki step for now.

Reply:
Removed it from loki, I think this was there just for testing. Now we have a place to use this step, added it in https://github.com/openshift/release/blob/6833bc03e0bbc6832f8893456b734557fa4be5f2/ci-operator/step-registry/openshift/e2e/aws/ccm/openshift-e2e-aws-ccm-workflow.yaml